plugins/missions.js

```diff
@@ -904,6 +1045,24 @@ window.plugin.missions = {
 		}
 	},
 
+	onMoveEnd: function()
+	{
+		// When autorefresh is enabled (currently always)
+		// and not showing mission list of one portal
+		// and window not collpased
+		if(this.autoRefreshOnMoveEnd && !this.isShowingPortalList && !this.isMissionListCollapsed)
+		{
+			let dialog = window.DIALOGS['dialog-missionsList'];
```

Comment:
Use `var` https://github.com/IITC-CE/ingress-intel-total-conversion/wiki/Code-Style

And why not  `window.DIALOGS.dialog-missionsList`? Do we have some convention on such usage? (just a question, not request to change)

Reply:
Minus is not allowed in identifiers.